cmake/FindFreeimage.cmake

```diff
@@ -4,7 +4,14 @@ include (${gazebo_cmake_dir}/GazeboUtils.cmake)
 ########################################
 # Find packages
 if (PKG_CONFIG_FOUND)
-  pkg_check_modules(freeimage freeimage>=${MIN_FREEIMAGE_VERSION})
+  find_package(freeimage CONFIG)
+  if (freeimage_FOUND)
+    set(freeimage_LIBRARIES freeimage::FreeImage)
+    message (STATUS "FreeImage found")
+  endif()
```

Comment:
Can you move this block after the `endif (PKG_CONFIG_FOUND)` block? In this way we ensure that nothing at all changes for setups that find freeimage with pkg-config, and it is also more logical that we call `find_package(freeimage CONFIG)`  even if pkg-config is not installed.

Reply:
I will try.

Reply:
done.